tasks/uk/hierarchy.py

```diff
@@ -21,7 +21,8 @@
     'datashare_pa_geo': PostcodeAreas
 }
 
-LEVELS = list(LEVEL_CLASSES.keys())
+# LEVEL_CLASSES.keys() can't be used because it doesn't guarantee order
+LEVELS = ['cdrc_the_geom', 'odl_ps_geo', 'gov_lsoa_geo', 'gov_msoa_geo', 'odl_pd_geo', 'datashare_pa_geo']
```

Comment:
You could use an `OrderedDict`

Reply:
Indeed (I realized about it afterward and forgot to fix it). On it!